wcomponents-test-lib/src/main/java/com/github/bordertech/wcomponents/test/selenium/by/ByButtonImage.java

```diff
@@ -0,0 +1,24 @@
+package com.github.bordertech.wcomponents.test.selenium.by;
+
+import java.util.List;
+import org.openqa.selenium.By;
+import org.openqa.selenium.SearchContext;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.internal.FindsByClassName;
+
+/**
+ * Finds a button by the image on the button.
+ */
+public class ByButtonImage extends By {
+
+	/**
+	 * Css class to find the WTabSet.
```

Comment:
CSS not Css and there is a copy-paste error in this comment. You are not finding a WTabSet.